style/scss/_details.scss

```diff
@@ -0,0 +1,476 @@
+.main {
+  box-shadow: 0px 1px 18px 0px rgba($dark-gray, 0.35);
+  margin: 0 20px;
+  border-radius: 30px;
+  padding-bottom: 30px;
+}
+
+.back__wrapper {
+  margin: 140px 20px 20px 20px;
+}
+
+.backBtn {
+  padding-top: 15px;
+  display: flex;
+  align-items: center;
+
+  .backTxt {
+    font-weight: 500;
+    line-height: 24px;
+    font-size: 1.6rem;
+    color: $dark-gray;
+  }
+}
+
+.jd__container {

```

Comment:
You should use SASS Nesting classes advantages over CSS.
Example:

.card {
  // card styles here...

  &__heading { **this is .card__heading class - we can use this in jd__header, jd__main...etc**
    // card heading styles here
  }

  &__button {
    // card button styles here...

    &:focus-visible {
      // card button's keyboard focus styles here
    }
  }
}